qoffer-infrastructure/src/main/groovy/life/qbic/Presenter.groovy

```diff
@@ -0,0 +1,21 @@
+package life.qbic
+
+import life.qbic.portal.portlet.customers.create.CreateCustomerOutput
+import life.qbic.portal.portlet.customers.update.UpdateCustomerOutput
+import life.qbic.portal.portlet.offers.create.CreateOfferOutput
+import life.qbic.portal.portlet.offers.search.SearchOffersOutput
+import life.qbic.portal.portlet.offers.update.UpdateOfferOutput
+
+/**
+ *  Handles the presentation of the qoffer-2.0 use cases and its internal data model to the viewModel.
+ *
+ *  This class acts as connector between the use cases of qoffer-2.0 and its viewModel
+ *  and should be used to transfer data between these components.
+ *  It prepares the view model, which can then be displayed by the view implementing {@link PortletView}.
```

Comment:
Nit:
As far as I understood it the presenter does the conversion of the output of the use case into data that is stored in the view model. The `prepares the view model` sounds like the view model is expected to do something other than hold the data.

Reply:
Absolutely Correct and is written more explicitly